src/hb-ot-layout-gsubgpos.hh

```diff
@@ -3988,7 +3988,7 @@ struct GSUBGPOSVersion1_2
 {
   friend struct GSUBGPOS;
 
-  protected:
+  public: // TODO
```

Comment:
Add a friend statement if you can. Otherwise fine with me.

Reply:
Ended up just adding a method to access the field I needed.